dev-client/src/model/utils/promiseTimeoutUtil.ts

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
this is very nitpick, but this code isn't really related to the app's models, so it might make more sense outside of that folder